update.py

```diff
@@ -63,10 +63,12 @@ def check_notices(session, sessionData):
             notice['attachment_url'] = ERP_ATTACHMENT_URL + m.group(1)
             r = session.get(notice['attachment_url'], stream=True)
             r.raw.decode_content = True
-            hash_ = hashlib.md5()
-            for chunk in iter(lambda: r.raw.read(4096), b""):
-                hash_.update(chunk)
-            notice['attachment_md5'] = hash_.hexdigest()
+            notice['attachment_raw'] = r
```

Comment:
We weren't storing the raw attachment because over time this would consume all our storage (hence md5-ing it and storing just that).

A better approach here is to keep the md5, and ignore the `attachment_raw` field when doing db writes/reads.

Reply:
Hmm, but wouldn't we actually require the raw attachment for directly using from the mailer (in `hooks.py`) rather than the md5?

Reply:
Yes, you could pass `attachment_raw` through to the mailer, but remove it when interacting with the database. There are two interactions: the first when we search for any notices that match this one, and the second when we write a new notice into the database. We basically want to skip over the `attachment_raw` key for both these interactions (you can do this by constructing a new dict with only the required keys).